api/src/integrations/epic/base-record-utils.js

```diff
@@ -231,6 +231,36 @@ class BaseRecordUtils {
       defaultLog.error(`Failed to create ${this.recordType._schemaName} record: ${error.message}`);
     }
   }
+
+  /**
+   * Indicates if a record is a fee order or not.
+   * 
+   * @param {object} transformedRecord Epic record that has been transformed to NRPTI format
+   * @returns {boolean} Indication if the record is a fee order
+   * @memberof BaseRecordUtils
+   */
+  isRecordFeeOrder(transformedRecord) {
+    if (!transformedRecord || !transformedRecord.recordName) {
```

Comment:
More of an epic data question: do we know for sure that all records will have a recordName? Or is it ever acceptable to not have a name (in which case we shouldn't throw an error here)?

Reply:
It takes in a transformed record which guarantees that there is a recordName property. If you feed it a non-transformed record it will not have this property and that is when it should error.

Reply:
```!transformedRecord.recordName``` will trigger on an empty string though, which is the default recordName if nothing comes from Epic

Reply:
Good point. Updated to look for the actual object property rather than a falsey value.